README.md

```diff
@@ -17,18 +17,104 @@ Installation is easily done via `pip`. The package is then used as `optics_funct
 pip install optics_functions
 ```
 
-Example:
+## Documentation
+
+- Autogenerated docs via `Sphinx` can be found at <https://pylhc.github.io/optics_functions/>.
+- General documentation of the OMC-Team is located at <https://pylhc.github.io/>
+
+## Description
+
+This package serves as a library of functions to calculate various optics parameters such as RDTs and coupling from a MAD-X twiss output.
+The functionality mainly manipulates and returns TFS files or `TfsDataFrame` objects from the `tfs-pandas` package.
+
+
+### Usage Examples 
+Coupling Example:
 
 ```python
-import optics_functions
+import logging
+import sys
+
+import tfs  # tfs-pandas
+
+from optics_functions.coupling import coupling_from_cmatrix, closest_tune_approach
+from optics_functions.utils import split_complex_columns
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO, format="%(message)s")
+
+# read MAD-X twiss output
+df_twiss = tfs.read("twiss.tfs", index="NAME")
+
+# calculate coupling from the cmatrix
+df_coupling = coupling_from_cmatrix(df_twiss)
+
```

Comment:
add as comment the output of printing the df?

Reply:
```
                           F0002  ...               F2000
NAME                              ...                    
IP3           2.673376-1.045712j  ... -2.863617-0.789910j
MCBWV.4R3.B1  2.475684-1.453081j  ... -1.927365-2.260426j
BPMW.4R3.B1   2.470411-1.462027j  ... -1.862287-2.314336j
MQWA.A4R3.B1  2.440763-1.511004j  ... -1.413706-2.612603j
MQWA.B4R3.B1  2.228282-1.555324j  ... -0.788608-2.855177j
...                          ...  ...                 ...
MQWB.4L3.B1   2.733194+0.167312j  ... -2.632290+0.135418j
MQWA.B4L3.B1  2.763986-0.041253j  ... -2.713212+0.063256j
MQWA.A4L3.B1  2.804960-0.235493j  ... -2.847616-0.017922j
BPMW.4L3.B1   2.858218-0.266543j  ... -2.970384-0.032890j
MCBWH.4L3.B1  2.831426-0.472735j  ... -2.966818-0.149180j
```

That is very input specific...

Reply:
open for suggestions